recipes/opsin/2.1.0/meta.yaml

```diff
@@ -6,7 +6,7 @@ package:
     name: opsin
     version: 2.1.0
 build:
-  number: 0
+  number: 1
```

Comment:
The possibility to add notes to packages that are displayed in the package view on the homepage was introduced after most of the other java tools (see [GUIDELINES.md, first section](https://github.com/bioconda/bioconda-recipes/blob/master/GUIDELINES.md)). Ideally, all java tools have a note explaining the wrapper, because the manual of these tools will usually talk about jar based  invocation only.